frontend/src/app/login/authenticated.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="authentication">
+  <ng-container *ngIf="!isAuthenticated">
+    <a href="https://fs-intake-api-staging.app.cloud.gov/auth/login-gov/openid/login">Login</a>
```

Comment:
Going forward, I'd rather not have partially finished components/stories committed to the master branch. To me that's not a particularly good practice and it should be a bit easier to do with another instance of the app spun up.